src/AppInstallerCommonCore/Public/AppInstallerDeployment.h

```diff
@@ -7,6 +7,13 @@
 
 namespace AppInstaller::Deployment
 {
+    // Calls winrt::Windows::Management::Deployment::PackageManager::AddPackageAsync
+    // which does not trigger smart screen scan
+    void AddPackage(

```

Comment:
>AddPackage [](start = 9, length = 10)

It would be better to have a single function in our code that takes in an additional bool for whether SmartScreen should be called.  Having every call site need to have a `if (trust) call one; else call other;` is overly verbose, and potentially leads to bugs.

Think about the next time someone adds an AddPackage type call, and whether they will remember to call both depending on some trust level.  If the function demands the parameter, they will stop and think about it and make a choice.  If comes down to which function they call, they may not be aware of the choice at all.